apps/token-e2e/src/support/governance.functions.js

```diff
@@ -87,6 +87,7 @@ Cypress.Commands.add(
 
 Cypress.Commands.add('get_proposal_id_from_list', (proposalTitle) => {
   cy.contains(proposalTitle)
+    .parent()
```

Comment:
Question (non-blocking): are you able to use [parentsUntil](https://docs.cypress.io/api/commands/parentsuntil) instead? Seem like it'd be less brittle

Reply:
Thanks Sam, I didn't know about that one! And I agree parentsUntil is much better here. 
Updated